### PR TITLE
[Naitei] Share_Product_User

### PR DESCRIPTION
**Pull request by ThaiKhacDuoc**
## Related Tickets
- ticket redmine

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

https://github.com/user-attachments/assets/65949e84-b388-45e7-a83f-e9398f047a78

### CODE REVIEW

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/views/devise/passwords/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<h2>Change your password</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <%= f.hidden_field :reset_password_token %>
+
+  <div class="field">
+    <%= f.label :password, "New password" %><br />
+    <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em><br />
```
Comment (ghost) [inline app/views/devise/passwords/edit.html.erb]: i18n ddi em

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -15,6 +15,7 @@
     <%= stylesheet_link_tag "cart", "data-turbo-track": "reload" %>
     <%= stylesheet_link_tag "bill", "data-turbo-track": "reload" %>
     <%= stylesheet_link_tag "https://cdnjs.cloudflare.com/ajax/libs/toastr.js/2.1.3/toastr.min.css" %>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
```
Comment (ghost) [inline app/views/layouts/application.html.erb]: dùng import map đi em

**Comment by ThaiKhacDuoc** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**